CVChatbot/CVChatbot.Bot/ChatbotActions/Commands/Stats.cs

```diff
@@ -17,9 +17,9 @@ public override void RunAction(ChatExchangeDotNet.Message userMessage, ChatExcha
 
             var statsTable = doc.DocumentNode
                 .Descendants("table")
-                .Where(x => x.Attributes["class"] != null)
-                .Where(x => x.Attributes["class"].Value == "task-stat-table")
-                .Single();
+                .Single(x => 
+                    x.Attributes["class"].Value == "task-stat-table" && 
+                    x.Attributes["class"] != null);
```

Comment:
Yes, as there's less overhead with just calling `Single` vs. calling `Where` twice. I'd have to do some benchmarks for any more info.